src/classes/Wcms.php

```diff
@@ -1323,4 +1323,4 @@ private function zipBackup(): void
         $zip->close();
         $this->redirect('data/files/' . $zipName);
     }
-}
+}
```

Comment:
It's good practice to have a newline at end of file ;)

Reply:
@NicolasCARPi thank you, always wondered why GitHub added this automatically. Mystery solved!